docs/content/operations/api-reference.md

```diff
@@ -0,0 +1,426 @@
+---
+layout: doc_page
+---
+
+# API Reference
+
+This page documents all of the API endpoints for each Druid service type.
+
+## Table of Contents
```

Comment:
What about including a link for 'Common' to the table of contents too for consistency?

Reply:
Added a link to Common